src/bin/util/test/mldsa44-import-key-test.sh

```
@@ -0,0 +1,158 @@
+*"OpenSSL 3.1."*|\
+*"OpenSSL 3.2."*|\
+*"OpenSSL 3.3."*|\
+*"OpenSSL 3.4."*)
+  echo "$openssl_version is not impacted" >&2
+  exit 77
+  ;;
+esac
+# NOTE OpenSSL > 3.5
+
+if test "$RUNNER_OS" = "Windows" ; then
+  D=`cd ../../../lib/Debug/ && pwd`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)